spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/error/AbstractErrorController.java

```diff
@@ -62,7 +63,7 @@ public AbstractErrorController(ErrorAttributes errorAttributes, List<ErrorViewRe
 
 	private List<ErrorViewResolver> sortErrorViewResolvers(List<ErrorViewResolver> resolvers) {
 		List<ErrorViewResolver> sorted = new ArrayList<>();
-		if (resolvers != null) {
```

Comment:
There's no guarantee that resolvers is not null since there is no assertion in the constructor.